swagger.yaml

```diff
@@ -183,8 +185,12 @@ paths:
           type: string
         - name: source_metadata
           in: query
-          description: '???'
+          description: >-
+            A JSON blob containing extra source specific information.  This
+            field is free-form, but we generally expect the data will follow
+            one of the formats outlined in the `/schema` folder.
```

Comment:
I think it would be helpful if `/schema` were a link to the actual JSON file in question.